### ISSUE TITLE
Update all grease pencil descriptions - chapter 7.3.14

### ISSUE THREAD

**Issue by ReinerBforartists** [Bforartists/Manual#678]
With GP3 we got new brushes with new functionality. The old brushes descriptions are not longer valid and needs an update.

This task is for chapter 7.3.14

Related: https://github.com/Bforartists/Manual/issues/674

**Comment by ReinerBforartists** [comment]
Hm, this chapter is up to date. Closing.

**Comment by ReinerBforartists** [comment]
Ah, wait, the single brushes brush settings are not documented.

**Comment by Draise14** [comment]
I am wondering if the single brush settings should be in the Essentials brush chapter, since they are now essential brush assets - not brush tools anymore.

We could document the general settings, but these are not brush asset agnostic.

**Comment by ReinerBforartists** [comment]
Hmm, makes me wonder if the essential brush chapter make sense at all.

**Comment by Draise14** [comment]
Right now the essentials chapter covers the geometry nodes groups, which I pulled from the geometry nodes sections since they are now assets.

**Comment by Draise14** [comment]
For the sake of brushes though, yeah, document their features, types, and features - but they are now user assets so "per brush" shouldn't be that heavily documented. 

The Default Asset library chapter and Essential assets chapter covers the surface of an asset, but not in depth in a GUI level.

**Comment by ReinerBforartists** [comment]
Yeah, you can do that. But logically it is now harder to find. When you touch the node editor then you expect to find the node related stuff in the node chapters. This is what worries me a bit here with the brushes now. You are in draw mode, in the brush settings, and find the brush settings explanation then in a completely different chapter?

**Comment by Draise14** [comment]
Problem is.. assets are located in the asset browser and asset shelves which show accross multiple editors now. 

So.. it's not locked to a certain area. 

Texture brushes now show in Asset Browser, Image Editor - Paint mode and 3D View Paint Mode Asset shelf. 3 locations. Do we repeat ourselves 3 times?

**Comment by ReinerBforartists** [comment]
Yes, i completely understand. But does the user also understand? I am already in trouble as the author where to put the information then. It breaks our rule of having the stuff explained in the corresponding ui area chapters.

The question to repeat ourselves was once answered with a yes, if necessary. Because we have this strict rule that the things needs to be explained where it belongs. we are in the tool shelf? then explain it in the tool shelf chapter. Even when the tools are the same across three modes. And same with brushes.

**Comment by Draise14** [comment]
Righteo, so repetition is allowed, guess that makes sense for searcheability.

**Comment by ReinerBforartists** [comment]
Indeed :)

I know it is difficult. But as a user you might not know that the same functionality is also in another mode :)

**Comment by ReinerBforartists** [comment]
Let's have a look here. Description for five brushes to add ...
